### PR TITLE
Merge 2.19 final to trunk

### PR DESCRIPTION
**Pull request by oguzkocer**

### CODE REVIEW

**Comment by AliSoftware** [comment]
⚠️ @oguzkocer wrong target branch for this PR?

**Comment by oguzkocer** [comment]
@AliSoftware Yes, sorry 🤦 Should be fixed now.

File: Simplenote/build.gradle [hunk 1]
```diff
@@ -30,9 +30,9 @@ android {
         if (project.hasProperty("versionName")) {
             versionName project.property("versionName")
         } else {
-            versionName "2.18"
+            versionName "2.19"
         }
-        versionCode 138
+        versionCode 140
```
Comment (AliSoftware) [inline Simplenote/build.gradle:35]: Unsure why the jump, but 🤷

File: fastlane/Pluginfile [hunk 2]
```diff
@@ -2,4 +2,4 @@
 #
 # Ensure this file is checked in to source control!
 
-gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', tag: '0.13.0'
+gem 'fastlane-plugin-wpmreleasetoolkit', '~> 1.3.1'
```
Comment (AliSoftware) [inline fastlane/Pluginfile:5]: Side note: you might want to update to `~> 1.4` (and _not_ `~> 1.4.0`, btw) in `develop` at some point during next sprint 🙂

**Review by AliSoftware — APPROVED** [review]
Branch graph makes sense and version bumps seems good too, ready to ship!

~_(not sure if you prefer waiting for Google approval on monday to merge or not, so will let you push the button)_~ eh you enabled auto-merge after all 👍

**Pull request merged**